VoIPController.cpp

```diff
@@ -3005,7 +3005,7 @@ static void initMachTimestart() {
 #endif
 
 double VoIPController::GetCurrentTime(){
-#if defined(__linux__)
+#if defined(__linux__) || defined(__FreeBSD__)
```

Comment:
is this linux/freebsd specific? Maybe there should be used some __UNIX__?

Reply:
The same question applies for all other similar places

Reply:
clock_gettime() is POSIX. For some reason unknown to me, libtgvoip code decides to obtain time on Apple differently to Linux. @grishka ?

Using _POSIX_VERSION here would disable compiling the latter code on Apple. Using __unix__ (lowercase) would enable it on Linux and FreeBSD, not enabling on Apple, so would also work. I personally prefer more explicit list here, but if you prefer __unix__, I can change to that.

Reply:
Huh, someone still uses this library? I don't really remember why I did this, but Mach (Apple kernel) isn't always 100% compatible with Linux in terms of syscalls.

Reply:
I think this explains https://stackoverflow.com/questions/41509505/clock-gettime-on-macos